integration-cli/check_test.go

```diff
@@ -189,18 +188,18 @@ func (s *DockerDaemonSuite) TearDownTest(c *check.C) {
 }
 
 func (s *DockerDaemonSuite) TearDownSuite(c *check.C) {
-	err := filepath.Walk(daemonSockRoot, func(path string, fi os.FileInfo, err error) error {
+	filepath.Walk(daemonSockRoot, func(path string, fi os.FileInfo, err error) error {
 		if err != nil {
-			return err
+			// ignore errors here
+			// not cleaning up sockets is not really an error
+			return nil
```

Comment:
Realistically, if this was a problem, it would show up in other ways (like all the daemon tests failing).